tests/integration/create_get_delete/integration_test.go

```diff
@@ -73,31 +93,37 @@ var _ = Describe("Create (Integration)", func() {
 		)
 
 		It("should not return an error", func() {
-			if skipCreation {
-				fmt.Printf("Creation test skip: %t\n", skipCreation)
+			if !doCreate {
+				fmt.Printf("will use existing cluster %s", clusterName)
 				return
 			}
 
-			args := []string{"create", "cluster", "-n", clusterName, "-t", "t2.medium", "-N", "1", "-r", eksRegion, "--kubeconfig", pathToEksCtl}
+			fmt.Printf("Using kubeconfig: %s\n", kubeconfigPath)
+
+			if clusterName == "" {
```

Comment:
Do we need to allow this customisation/auto-generation for an integration test? I always thought it would be hardcoded.

Reply:
Can always change this later. I thought hardcoded one is good for `integration-test-dev` where it doesn't get re-created/destroyed...